spec/features/logs_spec.rb

```diff
@@ -279,7 +277,7 @@
           # negative (not failing). A wait time of 1 second seems to be long
           # enough to make the test consistently fail if this is broken.
           publish_new_log_entry
-          sleep(1)
+          sleep(0.5)
```

Comment:
I'm leaving the comment mentioning that a 1 second sleep is reliable, but I'm decreasing this to 0.5 seconds to be faster but hopefully still be able to catch a bug at least intermittently if there is one.